2022/06/04 18:53:10 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: 2022-06-04T18:53:10+00:00 [critical] Uncaught Exception: Unable to write in the "logs" directory (/var/www/var/log)" while reading upstream, client: 172.20.0.1, server: symfony.loc, request: "GET /index.php HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1"
2022/06/04 18:53:12 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: 2022-06-04T18:53:12+00:00 [critical] Uncaught Exception: Unable to write in the "logs" directory (/var/www/var/log)" while reading upstream, client: 172.20.0.1, server: symfony.loc, request: "GET /index.php HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1"
2022/06/04 18:53:13 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: 2022-06-04T18:53:13+00:00 [critical] Uncaught Exception: Unable to write in the "logs" directory (/var/www/var/log)" while reading upstream, client: 172.20.0.1, server: symfony.loc, request: "GET /index.php HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1"
2022/06/04 18:53:14 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: 2022-06-04T18:53:14+00:00 [critical] Uncaught Exception: Unable to write in the "logs" directory (/var/www/var/log)" while reading upstream, client: 172.20.0.1, server: symfony.loc, request: "GET /index.php HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1"
2022/06/04 18:53:19 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: 2022-06-04T18:53:19+00:00 [critical] Uncaught Exception: Unable to write in the "logs" directory (/var/www/var/log)" while reading upstream, client: 172.20.0.1, server: symfony.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1"
2022/06/04 18:53:50 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: 2022-06-04T18:53:50+00:00 [critical] Uncaught Exception: Unable to write in the "logs" directory (/var/www/var/log)" while reading upstream, client: 172.20.0.1, server: symfony.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1"
2022/06/04 18:53:52 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: 2022-06-04T18:53:52+00:00 [critical] Uncaught Exception: Unable to write in the "logs" directory (/var/www/var/log)" while reading upstream, client: 172.20.0.1, server: symfony.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1"
2022/06/04 18:53:52 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: 2022-06-04T18:53:52+00:00 [critical] Uncaught Exception: Unable to write in the "logs" directory (/var/www/var/log)" while reading upstream, client: 172.20.0.1, server: symfony.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1"
2022/06/04 18:53:53 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: 2022-06-04T18:53:53+00:00 [critical] Uncaught Exception: Unable to write in the "logs" directory (/var/www/var/log)" while reading upstream, client: 172.20.0.1, server: symfony.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1"
2022/06/04 18:53:54 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: 2022-06-04T18:53:54+00:00 [critical] Uncaught Exception: Unable to write in the "logs" directory (/var/www/var/log)" while reading upstream, client: 172.20.0.1, server: symfony.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1"
2022/06/04 18:53:56 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: 2022-06-04T18:53:56+00:00 [critical] Uncaught Exception: Unable to write in the "logs" directory (/var/www/var/log)" while reading upstream, client: 172.20.0.1, server: symfony.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1"
2022/06/04 18:53:56 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: 2022-06-04T18:53:56+00:00 [critical] Uncaught Exception: Unable to write in the "logs" directory (/var/www/var/log)" while reading upstream, client: 172.20.0.1, server: symfony.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1"
2022/06/04 18:53:57 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: 2022-06-04T18:53:57+00:00 [critical] Uncaught Exception: Unable to write in the "logs" directory (/var/www/var/log)" while reading upstream, client: 172.20.0.1, server: symfony.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1"
2022/06/04 18:53:58 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: 2022-06-04T18:53:58+00:00 [critical] Uncaught Exception: Unable to write in the "logs" directory (/var/www/var/log)" while reading upstream, client: 172.20.0.1, server: symfony.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1"
2022/06/04 18:53:59 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: 2022-06-04T18:53:59+00:00 [critical] Uncaught Exception: Unable to write in the "logs" directory (/var/www/var/log)" while reading upstream, client: 172.20.0.1, server: symfony.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1"
2022/06/04 18:54:00 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: 2022-06-04T18:54:00+00:00 [critical] Uncaught Exception: Unable to write in the "logs" directory (/var/www/var/log)" while reading upstream, client: 172.20.0.1, server: symfony.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1"
2022/06/04 18:54:01 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: 2022-06-04T18:54:01+00:00 [critical] Uncaught Exception: Unable to write in the "logs" directory (/var/www/var/log)" while reading upstream, client: 172.20.0.1, server: symfony.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1"
2022/06/04 18:54:02 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: 2022-06-04T18:54:02+00:00 [critical] Uncaught Exception: Unable to write in the "logs" directory (/var/www/var/log)" while reading upstream, client: 172.20.0.1, server: symfony.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1"
2022/06/04 18:54:03 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: 2022-06-04T18:54:03+00:00 [critical] Uncaught Exception: Unable to write in the "logs" directory (/var/www/var/log)" while reading upstream, client: 172.20.0.1, server: symfony.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1"
2022/06/04 18:54:03 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: 2022-06-04T18:54:03+00:00 [critical] Uncaught Exception: Unable to write in the "logs" directory (/var/www/var/log)" while reading upstream, client: 172.20.0.1, server: symfony.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1"
2022/06/04 18:55:12 [error] 28#28: *22 FastCGI sent in stderr: "PHP message: 2022-06-04T18:55:12+00:00 [critical] Uncaught Exception: Unable to write in the "logs" directory (/var/www/var/log)" while reading upstream, client: 172.20.0.1, server: symfony.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1"
2022/06/04 19:03:23 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: 2022-06-04T19:03:23+00:00 [critical] Uncaught Exception: Unable to write in the "logs" directory (/var/www/var/log)" while reading upstream, client: 172.20.0.1, server: symfony.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1"
2022/06/04 19:03:25 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: 2022-06-04T19:03:25+00:00 [critical] Uncaught Exception: Unable to write in the "logs" directory (/var/www/var/log)" while reading upstream, client: 172.20.0.1, server: symfony.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1"
2022/06/04 19:03:25 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: 2022-06-04T19:03:25+00:00 [critical] Uncaught Exception: Unable to write in the "logs" directory (/var/www/var/log)" while reading upstream, client: 172.20.0.1, server: symfony.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1"
2022/06/04 19:03:26 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: 2022-06-04T19:03:26+00:00 [critical] Uncaught Exception: Unable to write in the "logs" directory (/var/www/var/log)" while reading upstream, client: 172.20.0.1, server: symfony.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1"
2022/06/04 19:03:27 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: 2022-06-04T19:03:27+00:00 [critical] Uncaught Exception: Unable to write in the "logs" directory (/var/www/var/log)" while reading upstream, client: 172.20.0.1, server: symfony.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1"
2022/06/04 19:03:28 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: 2022-06-04T19:03:28+00:00 [critical] Uncaught Exception: Unable to write in the "logs" directory (/var/www/var/log)" while reading upstream, client: 172.20.0.1, server: symfony.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1"
2022/06/04 19:13:06 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: 2022-06-04T19:13:06+00:00 [critical] Uncaught Exception: Unable to write in the "logs" directory (/var/www/var/log)" while reading upstream, client: 172.20.0.1, server: symfony.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1"
2022/06/04 19:40:02 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: 2022-06-04T19:40:02+00:00 [critical] Uncaught Exception: Unable to write in the "logs" directory (/var/www/var/log)" while reading upstream, client: 172.20.0.1, server: symfony.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1"
2022/06/04 19:40:04 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: 2022-06-04T19:40:04+00:00 [critical] Uncaught Exception: Unable to write in the "logs" directory (/var/www/var/log)" while reading upstream, client: 172.20.0.1, server: symfony.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1"
2022/06/04 19:40:05 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: 2022-06-04T19:40:05+00:00 [critical] Uncaught Exception: Unable to write in the "logs" directory (/var/www/var/log)" while reading upstream, client: 172.20.0.1, server: symfony.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1"
2022/06/04 19:40:06 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: 2022-06-04T19:40:06+00:00 [critical] Uncaught Exception: Unable to write in the "logs" directory (/var/www/var/log)" while reading upstream, client: 172.20.0.1, server: symfony.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1"
2022/06/04 19:40:44 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: 2022-06-04T19:40:44+00:00 [critical] Uncaught Exception: Unable to write in the "logs" directory (/var/www/var/log)" while reading upstream, client: 172.20.0.1, server: symfony.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1"
2022/06/04 19:40:45 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: 2022-06-04T19:40:45+00:00 [critical] Uncaught Exception: Unable to write in the "logs" directory (/var/www/var/log)" while reading upstream, client: 172.20.0.1, server: symfony.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1"
2022/06/04 19:40:46 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: 2022-06-04T19:40:46+00:00 [critical] Uncaught Exception: Unable to write in the "logs" directory (/var/www/var/log)" while reading upstream, client: 172.20.0.1, server: symfony.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1"
2022/06/04 19:40:47 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: 2022-06-04T19:40:47+00:00 [critical] Uncaught Exception: Unable to write in the "logs" directory (/var/www/var/log)" while reading upstream, client: 172.20.0.1, server: symfony.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1"
2022/06/04 19:40:50 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: 2022-06-04T19:40:50+00:00 [critical] Uncaught Exception: Unable to write in the "logs" directory (/var/www/var/log)" while reading upstream, client: 172.20.0.1, server: symfony.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1"
2022/06/04 19:40:51 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: 2022-06-04T19:40:51+00:00 [critical] Uncaught Exception: Unable to write in the "logs" directory (/var/www/var/log)" while reading upstream, client: 172.20.0.1, server: symfony.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1"
2022/06/04 19:40:52 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: 2022-06-04T19:40:52+00:00 [critical] Uncaught Exception: Unable to write in the "logs" directory (/var/www/var/log)" while reading upstream, client: 172.20.0.1, server: symfony.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1"
2022/06/04 19:40:53 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: 2022-06-04T19:40:53+00:00 [critical] Uncaught Exception: Unable to write in the "logs" directory (/var/www/var/log)" while reading upstream, client: 172.20.0.1, server: symfony.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1"
2022/06/04 19:40:59 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: 2022-06-04T19:40:59+00:00 [critical] Uncaught Exception: Unable to write in the "logs" directory (/var/www/var/log)" while reading upstream, client: 172.20.0.1, server: symfony.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1"
2022/06/04 19:41:01 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: 2022-06-04T19:41:01+00:00 [critical] Uncaught Exception: Unable to write in the "logs" directory (/var/www/var/log)" while reading upstream, client: 172.20.0.1, server: symfony.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1"
2022/06/04 20:02:41 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: [error] Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://127.0.0.1/"" at /var/www/vendor/symfony/http-kernel/EventListener/RouterListener.php line 128" while reading response header from upstream, client: 172.20.0.1, server: symfony.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1"
2022/06/04 20:30:18 [error] 28#28: *7 FastCGI sent in stderr: "PHP message: [error] Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://127.0.0.1/"" at /var/www/vendor/symfony/http-kernel/EventListener/RouterListener.php line 128" while reading response header from upstream, client: 172.20.0.1, server: symfony.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1"
2022/06/04 21:50:27 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: [error] Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://127.0.0.1/"" at /var/www/vendor/symfony/http-kernel/EventListener/RouterListener.php line 128" while reading response header from upstream, client: 172.20.0.1, server: symfony.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1"
2022/06/04 21:50:28 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: [error] Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://127.0.0.1/"" at /var/www/vendor/symfony/http-kernel/EventListener/RouterListener.php line 128" while reading response header from upstream, client: 172.20.0.1, server: symfony.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1"
2022/06/04 21:50:29 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: [error] Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://127.0.0.1/"" at /var/www/vendor/symfony/http-kernel/EventListener/RouterListener.php line 128" while reading response header from upstream, client: 172.20.0.1, server: symfony.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1"
2022/06/04 21:50:30 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: [error] Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://127.0.0.1/"" at /var/www/vendor/symfony/http-kernel/EventListener/RouterListener.php line 128" while reading response header from upstream, client: 172.20.0.1, server: symfony.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1"
2022/06/04 21:50:31 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: [error] Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://127.0.0.1/"" at /var/www/vendor/symfony/http-kernel/EventListener/RouterListener.php line 128" while reading response header from upstream, client: 172.20.0.1, server: symfony.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1"
2022/06/04 21:50:32 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: [error] Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://127.0.0.1/"" at /var/www/vendor/symfony/http-kernel/EventListener/RouterListener.php line 128" while reading response header from upstream, client: 172.20.0.1, server: symfony.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1"
2022/06/04 21:50:32 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: [error] Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://127.0.0.1/"" at /var/www/vendor/symfony/http-kernel/EventListener/RouterListener.php line 128" while reading response header from upstream, client: 172.20.0.1, server: symfony.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1"
